index.php

```diff
@@ -36,11 +36,12 @@
 if (file_exists($maintenanceFile)) {
     $maintenanceBypass = false;
 
-    if (file_exists($maintenanceIpFile)) {
+    if (file_exists($maintenanceIpFile) && is_readable($maintenanceIpFile)) {
         /* if maintenanceFile and maintenanceIpFile are set use Mage to get remote IP (in order to respect remote_addr_headers xml config) */
         Mage::init($mageRunCode, $mageRunType);
         $currentIp = Mage::helper('core/http')->getRemoteAddr();
-        $allowedIps = explode(',', trim(file_get_contents($maintenanceIpFile)));
+        $allowedIps = preg_split('/[\ \n\,]+/', file_get_contents($maintenanceIpFile), 0, PREG_SPLIT_NO_EMPTY);
+        $allowedIps = array_map("trim", $allowedIps);
```

Comment:
do we need the trim since the split should remove double spaces?

Reply:
Yup indeed `trim()` can be....🥁...trimmed.